bot/resources/tags/dunder-methods.md

```diff
@@ -1,10 +1,12 @@
-### Dunder methods
+**Dunder methods**
 
 Double-underscore methods, or "dunder" methods, are special methods defined in a class that are invoked implicitly. Like the name suggests, they are prefixed and suffixed with dunders. You've probably already seen some, such as the `__init__` dunder method, also known as the "constructor" of a class, which is implicitly invoked when you instantiate an instance of a class.
 
-When you create a new class, all the default dunder methods are inherited from its superclass (which is `object` if no superclass is specified). However, we can override them by redefining the methods within this new class. For example, the default `__init__` method from `object` doesn't take any arguments, so we almost always override that to fit our needs.
+When you create a new class, all methods - including dunder methods - are inherited from its superclass, which is by default `object`. `object` has a set of default dunder methods, but we can override them by redefining the methods within this new class. For example, the default `__init__` method from `object` doesn't take any arguments, so we almost always override that to fit our needs.
```

Comment:
Sorry, I'm going to be a little nitpicky here. I think this can be worded a little better. At the moment, the first sentence is quiet long, with a lot of asides in it. We could probably get away with something much shorter. I don't think we need to really explain inheritance here. The first sentence attempts to do that a little, but I think we can get away with just stating that there are defaults that come from `object`.

Reply:
yep, resolved